### ISSUE TITLE
ModuleNotFoundError when trying to run eola

### ISSUE THREAD

**Issue by vegerot** [3b1b/manim#422]
While trying to run the first few chapters of eola by doing something like `python -m manim old_projects/eola/chapter0.py -pl`, I get an error saying
`ModuleNotFoundError: No module named 'once_useful_constructs'` 
`once_useful_constucts` does exist though in `manimlib/once_useful_constructs`. 
How do I get the module to work correctly in manim?

**Comment by Elteoremadebeethoven** [comment]
Change ```from once_useful_constructs import *``` for ```from manimlib.once_useful_constructs import *```.

**Comment by Elteoremadebeethoven** [comment]
Yes, since eola has been created a version of python 2, you have to translate to python 3. There are new commands to do this. I recommend you check the most recent projects for 3b1b.
